src/reconcile/rec_visibility.c

```diff
@@ -218,9 +211,8 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins,
 		if (upd_select->upd == NULL && r->las_skew_newest)
 			upd_select->upd = upd;
 
-		if ((F_ISSET(r, WT_REC_VISIBLE_ALL) ?
+		if (F_ISSET(r, WT_REC_VISIBLE_ALL) ?
 		    !__wt_txn_upd_visible_all(session, upd) :
-		    !__wt_txn_upd_visible(session, upd)) ||
 		    !__wt_txn_upd_durable(session, upd)) {
```

Comment:
I changed `__wt_txn_upd_visible_all` to respect durable timestamps, in part to simplify this condition.  (I also think it's required in other places).  It's probably time to discuss the `visible_all` naming...